src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,11 +6,10 @@
 public class CliSyntax {
 
     /* Prefix definitions */
-    public static final Prefix PREFIX_NAME = new Prefix("-n");
-    public static final Prefix PREFIX_PHONE = new Prefix("-p");
-    public static final Prefix PREFIX_EMAIL = new Prefix("-e");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("-a");
-    public static final Prefix PREFIX_TAG = new Prefix("-t");
-    public static final Prefix PREFIX_CATEGORY = new Prefix("-c");
-
+    public static final Prefix PREFIX_NAME = new Prefix("-n ");
+    public static final Prefix PREFIX_PHONE = new Prefix("-p ");
+    public static final Prefix PREFIX_EMAIL = new Prefix("-e ");
+    public static final Prefix PREFIX_ADDRESS = new Prefix("-a ");
+    public static final Prefix PREFIX_TAG = new Prefix("-t ");
+    public static final Prefix PREFIX_CATEGORY = new Prefix("-c ");
```

Comment:
```suggestion
    public static final Prefix PREFIX_NAME = new Prefix(" -n ");
    public static final Prefix PREFIX_PHONE = new Prefix(" -p ");
    public static final Prefix PREFIX_EMAIL = new Prefix(" -e ");
    public static final Prefix PREFIX_ADDRESS = new Prefix(" -a ");
    public static final Prefix PREFIX_TAG = new Prefix(" -t ");
    public static final Prefix PREFIX_CATEGORY = new Prefix(" -c ");
```
Lets prepend a space too

Reply:
I am afraid it still fails parseCommand_edit() since
Expected = ...address=123, Jurong West Ave 6, #08-111, tags=[], category=null}} and
Actual = ...address=123, Jurong West Ave 6, #08-111  -t, tags=null, category=null}}

Reply:
From few debugging runs I did, It seemed that the whitespace after -t is causing the issue as
it passes all when I run tests with 
```
public static final Prefix PREFIX_NAME = new Prefix("-n ");
public static final Prefix PREFIX_PHONE = new Prefix("-p ");
public static final Prefix PREFIX_EMAIL = new Prefix("-e ");
public static final Prefix PREFIX_ADDRESS = new Prefix("-a ");
public static final Prefix PREFIX_TAG = new Prefix("-t");
public static final Prefix PREFIX_CATEGORY = new Prefix("-c ");
```

Reply:
My bad, I meant
```java
final String arguments = String.format("%s ", matcher.group("arguments"));
```
in `AddressBookParser.java` is the fix for the failing testcase